pyproject.toml

```diff
@@ -39,6 +39,7 @@ python-magic = "^0.4.24"
 uwsgi = "^2.0.20"
 python-jenkins = "^1.7.0"
 python-slugify = "^6.1.1"
+pygithub = "^2.6.1"
```

Comment:
nice, does it support these big queries with paging?

Reply:
Exactly, you can set up page size on the entire client (100 max) and then every query that supports paging will use that size.